src/Explorer/QueryCopilot/Modal/WelcomeModal.tsx

```diff
@@ -0,0 +1,89 @@
+import { DefaultButton, IconButton, Modal } from "@fluentui/react";
+import { useBoolean } from "@fluentui/react-hooks";
+import React from "react";
+import Database from "../../../../images/CopilotDatabase.svg";
+import Flash from "../../../../images/CopilotFlash.svg";
+import Thumb from "../../../../images/CopilotThumb.svg";
+import CoplilotWelcomeIllustration from "../../../../images/CopliotWelcomeIllustration.svg";
+import "./WelcomeModal.css";
+
+export const WelcomeModal = ({ visible }: { visible: boolean }): JSX.Element => {
+  const [isModalVisible, { setFalse: hideModal }] = useBoolean(visible);
+  const disableModal = (): void => {
```

Comment:
Yeah I made it like so that unless you click the button it won't set it in local storage. If we want it to always set regardless of where you click it I'll switch to this approach.